### DISCUSSION TITLE
리뷰할 때

### DISCUSSION THREAD

**Discussion by dev-zoha** [lawnmowing-programmer/algo#58] [Q&A] [answered]
<img width="1191" alt="image" src="https://user-images.githubusercontent.com/126316476/222128156-1e8e2aac-cb3d-49dd-ab0d-77c8511f4c67.png">

아까 리뷰 다는 방법 파일체인지에서 할 수 있다고 보여주셨는데
전 리뷰어인데 왜 approve <- 이런게 안뜰까요

그냥 코멘트를 달았는데 pending...

**Comment by minjae9610** [comment, marked as answer]
지금처럼 여러 리뷰를 한번에 달게 되면 오른쪽 위에 초록색 버튼 눌러서 최종 승인을 해야 해요!

그리고, 지금 리뷰하시려고 하는 PR은 이미 브랜치 이름 컨벤션 문제로 이미 머지해버린 PR이에요 :(

**Reply by minjae9610** [reply]
![image](https://user-images.githubusercontent.com/33440010/222129472-1bd70c45-f5c8-4d12-b387-09909e560d86.png)
이 버튼이요!

**Comment by minjae9610** [comment]
질문은 카테고리를 QnA로 잡아주시면 좋아요 👍
